src/containsSyntaxErrors.js

```diff
@@ -6,7 +6,7 @@ export function containsSyntaxErrors(input) {
         `"${process.execPath}"`,
         [
             "--check",
-            input,
+            `"${input}"`,
```

Comment:
This seems to fix the bug that was reported, which might be caused by the fact that this is spawed with `shell: true`.